packages/plugin-web-attribution/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@amplitude/plugin-web-attribution",
```

Comment:
Even though the name implies this is Browser only I still think we should standardize Plugin naming to include the support platform (`-browser`) unless the plugin is platform agnostic.

Reply:
Sure, I will add it to the package name.